models/internal.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package models;
+option go_package = "github.com/flipkart-incubator/nexus/models";
+
+message NexusInternalRequest{
+  uint64 ID = 1;
+  bytes Req = 2;
+}
+
+message NodeInfo {
```

Comment:
Why is this type part of the internal proto. Given that we expose this type through the `ListMembers` grpc, shouldn't this be part of the nexus proto ?

Reply:
In order to avoid cyclic import.

`pkg/api/nexus.go`imports `github.com/flipkart-incubator/nexus/internal/raft`

And `ListMembers() (uint64, map[uint64]*models.NodeInfo)` function is declared at `pkg/api/nexus.go` and implemented at `internal/raft/replicator.go`